doc/source/drivers/raster/vrt.rst

```diff
@@ -1776,6 +1776,10 @@ with (:ref:`gdal_translate`).
 
 The effect of the ``nogcp`` option (added in GDAL 3.8) is to not copy the GCPs in the source dataset to the output dataset (:ref:`gdal_translate`).
 
+The effect of the ``epo`` option (added in GDAL 3.8) is that ``srcwin`` or ``projwin`` values that fall partially outside the source raster extent will be considered as an error as per (:ref:`gdal_translate`).
+
+The effect of the ``eco`` option (added in GDAL 3.8) is that ``srcwin`` or ``projwin`` values that fall completely outside the source raster extent will be considered as an error as per (:ref:`gdal_translate`).
+
```

Comment:
it is not obvious reading that (and this applies to nogcp as well) that the ``true`` value must be provided

Reply:
thank you, fixed now.   I'm intending to do a review of the whole set for consistency and grouping, it's probably good to do that soon.